index.html

```diff
@@ -1,59 +1,426 @@
+      <li class="icon"><a class="fa-brands fa-linkedin" href="#"></a> </li>
+      <li class="icon"><a class="fa-brands fa-medium" href="#"></a> </li>
+      <li class="icon"><a class="fa-brands fa-github" href="#"></a> </li>
+      <li class="icon"><a class="fa-brands fa-angellist" href="#"></a> </li>
+  </ul>
+  </div>
+  </div>
+</section>
+<!---down arrow-->
+<div class="downarrow">
+  <img src="image/Disabled.png" alt="downarrow">
+</div>
+
+<!---Recent work-->
+<section>
+  <div class="recentworkcontainer">
+    <div class="insideautorecent">
+      <div class="headwork">
+      <h1 class="titlerecent">My Recent Work</h1>
+    </div>
+  </div>
+  </div>
 
-    <nav class="mobileinfo">
-        <a class="mobilelogo"><h1>Welcome</h1></a>
+</section>
+
+<!--recentimage container-->
+<section class="recentimagecontainer">
+  <div class="insideautoimage">
+
+  <div class="imgpatholder">
+<div class="imagebox">
+  <img src="image/Img Plaholder.png" alt="placeholder" >
+
+</div>
+<div class="storiescontainer storiescontainer6">
+<div class="head3">
+  <h1>Multi-Post Stories Gain+Glory</h1>
+</div>
+<div class="insideautolanguage">
+<div class="language">
+  <ul>
+    <li><a href="#">Ruby on rails</a></li>
+
+        <li><a href="#">CSS</a></li>
+        <li><a href="#">Javascript</a></li>
+        <li><a href="#">HTML</a></li>
+  </ul>
+
+</div>
+</div>
+<div class="insideautobutton">
+<div class="seeproject">
+  <button>See Project</button>
+</div>
+</div>
+</div>
+  </div>
+</div>
+<br><br><br>
+
+  <div class="imagebox1">
+    <img src="image/Img Plaholder.png" alt="placeholder" >
+  
+  </div>
+  <div class="storiescontainer storiescontainer1">
+  <div class="head3">
+    <h1>Multi-Post Stories Gain+Glory</h1>
+  </div>
+  <div class="insideautolanguage">
+  <div class="language">
+    <ul>
+      <li><a href="#">Ruby on rails</a></li>
+  
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">Javascript</a></li>
+          <li><a href="#">HTML</a></li>
+    </ul>
+  
+  </div>
+  </div>
+  <div class="insideautobutton">
+  <div class="seeproject">
+    <button>See Project</button>
+  </div>
+  </div>
+  </div>
+    </div>
+  </div>
+
+    <br><br>
+
+    <div class="imagebox2">
+      <img src="image/Img Plaholder.png" alt="placeholder" >
+    
+    </div>
+    <div class="storiescontainer storiescontainer2">
+    <div class="head3">
+      <h1>Multi-Post Stories Gain+Glory</h1>
+    </div>
+    <div class="insideautolanguage">
+    <div class="language">
+      <ul>
+        <li><a href="#">Ruby on rails</a></li>
+    
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">Javascript</a></li>
+            <li><a href="#">HTML</a></li>
+      </ul>
+    
+    </div>
+    </div>
+    <div class="insideautobutton">
+    <div class="seeproject">
+      <button>See Project</button>
+    </div>
+    </div>
+    </div>
+      </div>
+    </div>
+
+      <br><br>
```

Comment:
[OPTIONAL] I add this as optional as it's not part of the requirements but it would improve your project 💯 I suggest you avoid using the `<br>` tag because it's not good practice to do so ❌ , you might be wondering why? 🤔 The main reason for not using `<br>` is that it's not semantic. If you want two items in different visual blocks, you probably want them in different logical blocks. In most cases this means just using different elements, for example `<p>Stuff</p><p>Other stuff</p>`, and then using CSS to space the blocks out properly. All of that to say that I suggest using CSS to space the elements instead of the `<br>` tag. 👍🏽